docs/outputs.rst

```diff
@@ -366,6 +376,48 @@ For CompCor decompositions, entries include:
     Entries that are not saved in the data file for denoising are still stored in metadata with the
     ``dropped`` prefix.
 
+.. caution::
+    Only a subset of these CompCor decompositions should be used for further denoising.
+    The original Behzadi aCompCor implementation [Behzadi2007]_ can be applied using
+    components from the combined masks, while the more recent Muschelli implementation
+    [Muschelli2014]_ can be applied using
+    the :abbr:`WM (white matter)` and :abbr:`CSF (cerebro-spinal fluid)` masks.
+    To determine the provenance of each component, consult the metadata file (described above).
+
+.. caution::
+    Similarly, if you are using anatomical or temporal CompCor it may not make sense
+    to use global signal regressors (``csf``, ``white_matter``, or ``global_signal``) -
+    see `#1049 <https://github.com/poldracklab/fmriprep/issues/1049>`_.
```

Comment:
Although I'd probably not recommend including both compartmental aCompCor time series and mean compartment (CSF, WM) time series in a single model, I don't agree that it doesn't make sense to use the overall global signal. In particular, @lindenmp and colleagues found in their evaluation that there can be an added benefit of GSR+aCompCor over and above aCompCor alone.

See the green aCompCor bars, particularly in B and C (fig. 1 from Parkes et al.):
<img src="https://user-images.githubusercontent.com/13735271/70570639-2416db00-1b51-11ea-8729-191903c07146.png" width="400" />

Reply:
/cc @satra